### PR TITLE
Feature/269

### PR DESCRIPTION
**Pull request by Louismaestre**

### CODE REVIEW

File: Application/mobile/app_code/lib/pages/container/container_state.dart [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        },
+      );
+      if (response.statusCode == 200) {
+        final Map<String, dynamic> responseData = json.decode(response.body);
+        final List<dynamic> containersData = responseData["container"];
+        setState(() {
+          containers = containersData.map((data) => ContainerList.fromJson(data)).toList();
+        });
+      } else {
+        print('Error getContainer(): ${response.statusCode}');
```
Comment (tbellicha) [inline Application/mobile/app_code/lib/pages/container/container_state.dart:42]: So, no information to the user that the request has failed ?

File: Application/mobile/app_code/lib/pages/container/container_state.dart [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      }
+    } catch (e) {
+      print('Error getContainer(): $e');
```
Comment (tbellicha) [inline Application/mobile/app_code/lib/pages/container/container_state.dart:45]: same here

File: Application/mobile/app_code/test/container_test.dart
Comment (31Nathan) [inline Application/mobile/app_code/test/container_test.dart:1]: image
Missing a lot of tests, please test your code

**Review by 31Nathan — CHANGES_REQUESTED** [review]

**Review by tbellicha — APPROVED** [review]

File: Application/server/server.js [hunk 3]
```diff
@@ -485,16 +485,6 @@ app.post('/api/user/password', async (req, res) => {
   }
 })
 
-app.get('/api/container/listall', async (req, res) => {
```
Comment (invpuppet) [inline Application/server/server.js:488]: where is that route now

**Pull request merged**